src/storm-pomdp-cli/storm-pomdp.cpp

```diff
@@ -143,347 +143,347 @@ namespace storm {
-                            search.getLastWinningRegion().print();
-                            std::cout << '\n';
-                        }
-                        if (qualSettings.isExportWinningRegionSet()) {
-                            std::size_t hash = pomdp.hash();
-                            search.getLastWinningRegion().storeToFile(qualSettings.exportWinningRegionPath(),
-                                                                      "model hash: " + std::to_string(hash));
-                        }
-
-                        search.finalizeStatistics();
-                        if (pomdp.getInitialStates().getNumberOfSetBits() == 1) {
-                            uint64_t initialState = pomdp.getInitialStates().getNextSetIndex(0);
-                            uint64_t initialObservation = pomdp.getObservation(initialState);
-                            // TODO this is inefficient.
-                            uint64_t offset = 0;
-                            for (uint64_t state = 0; state < pomdp.getNumberOfStates(); ++state) {
-                                if (state == initialState) {
-                                    break;
-                                }
-                                if (pomdp.getObservation(state) == initialObservation) {
-                                    ++offset;
-                                }
-                            }
-
-                            if (search.getLastWinningRegion().isWinning(initialObservation,
-                                                                        offset)) {
-                                STORM_PRINT_AND_LOG("Initial state is safe!\n");
-                            } else {
-                                STORM_PRINT_AND_LOG("Initial state may not be safe.\n");
-                            }
-                        } else {
-                            STORM_LOG_WARN("Output for multiple initial states is incomplete");
-                        }
-
-                        if (coreSettings.isShowStatisticsSet()) {
-                            STORM_PRINT_AND_LOG("#STATS Number of belief support states: "
-                                                        << search.getLastWinningRegion().beliefSupportStates() << '\n');
-                            if (qualSettings.computeExpensiveStats()) {
-                                auto wbss = search.getLastWinningRegion().computeNrWinningBeliefs();
-                                STORM_PRINT_AND_LOG(
-                                        "#STATS Number of winning belief support states: [" << wbss.first << "," << wbss.second
-                                                                                     << "]");
-                            }
-                            search.getStatistics().print();
-                        }
+            auto const& qualSettings = storm::settings::getModule<storm::settings::modules::QualitativePOMDPAnalysisSettings>();
```

Comment:
To ease further merges, I would suggest not to apply code formatting on storm-pomdp just now.

Reply:
it looks like there really are just whitespace changes in this file. I'm in favour of reverting this before merging